### ISSUE TITLE
Aquarium Server Error 

### ISSUE THREAD

**Issue by rychen58** [CIDARLAB/damplab-aquarium#7]
## Summary & Context

When technicians try to run any protocols in Aquarium, this error appears instead of the interactive protocol screens. 
![Aquarium Server Error](https://user-images.githubusercontent.com/32885235/75805058-14252680-5d4f-11ea-8712-c7176c2ab5fc.png)

## Expected Behavior

Normally, when technicians try to run protocols in Aquarium, an interactive screen that provides instructions to the technicians will appear. 

## Current Behavior

This error message is generated instead of protocols instructions.
![Aquarium Server Error](https://user-images.githubusercontent.com/32885235/75805312-9d3c5d80-5d4f-11ea-8e57-e046aead5d72.png)

## Possible Solution

Krishna suggested either restart the Database or Aquarium. 
Right now, Krishna has changed Aquarium to restart every day at midnight to resolve this problem. 

## Steps to Reproduce

1. Go to the Aquarium Manager Tab, start running a protocol.
2. The error appears instead of the instruction screen.

## Environment

Chrome

**Comment by rkrishnasanka** [comment]
So @bjkeller this is also an issue with too many connections getting generated and locking of the server

**Comment by bjkeller** [comment]
Restart the krill service

**Comment by rkrishnasanka** [comment]
Yeah, I just put in the cron job for restarting the thing. I just wanted to point out to you folks incase someone else has the same problem 😄

**Comment by bjkeller** [comment]
Was this resolved by the cron tab change?

This is a known issue

**Comment by rkrishnasanka** [comment]
This has not repeated yet, but I'm going to leave this issue active until the thing gets fixed eventually.
